pkg/controller/performanceprofile/status.go

```diff
@@ -144,3 +147,39 @@ func (r *ReconcilePerformanceProfile) getProgressingConditions(reason string, me
 		},
 	}
 }
+
+func (r *ReconcilePerformanceProfile) getConditionsByMCPStatus(profile *performancev1alpha1.PerformanceProfile) []conditionsv1.Condition {
```

Comment:
I think it more correct to name it, `getMCPConditionsByProfile` or just `getMCPConditions`